deps.edn

```diff
@@ -0,0 +1,39 @@
+{:paths   ["src/main" "resources"]
+
+ :deps    {org.clojure/clojure                 {:mvn/version "1.10.1"}
```

Comment:
This is waaaay more deps than are probably needed. semantic-ui-wrapper, garden-css, ghostwheel, datascript...oh wait, this was the demo I was using IN f3?

I don't think this is what I want to do with this repo....